packages/react-menu/src/components/Menu/useMenu.tsx

```diff
@@ -38,6 +52,9 @@ export const useMenu = (props: MenuProps, ref: React.Ref<HTMLElement>, defaultPr
     console.warn('Menu can only take one MenuTrigger and one MenuList as children');
   }
 
+  // TODO use Popper for the trigger ref
+  const triggerRef = React.useRef<HTMLElement>() as React.MutableRefObject<HTMLElement>;
+  state.triggerRef = (triggerRef as unknown) as React.MutableRefObject<HTMLElement>;
```

Comment:
```suggestion
  const triggerRef = React.useRef<HTMLElement>();
  state.triggerRef = triggerRef;
```

_nit_: It seems that on this stage type casts are redundant